interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,184 @@
                   </leafNode>
                 </children>
               </node>
+              <tagNode name="zone-cache">
+                <properties>
+                  <help>Load a zone into the recursor cache</help>
+                  <valueHelp>
+                    <format>txt</format>
```

Comment:
Missing constraint for the domain name.

Other places like system domain-name use:

```xml
      <leafNode name="domain-name" owner="${vyos_conf_scripts_dir}/system_host-name.py">
        <properties>
          <help>System domain name</help>
          <priority>6</priority>
          <constraint>
            <validator name="fqdn"/>
          </constraint>
        </properties>
      </leafNode>
```

Reply:
fixed